File: src/app/core/models/ChatModel.ts
import { MessageModel } from './MessageModel';

export class ChatModel {
    public SenderID: string;
    public SenderEmail: string;
    public WorkspaceId: string;
    public ChannelId: string;
    public ChatId: string;
    public Messages: MessageModel[];
    public Name: string;
}

File: WebAPI/AquaChatAPI.Domain/Models/ChatModel.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace AquaChatAPI.Domain.Models
{
    public class ChatModel
    {
        public string ChatId { get; set; }
        public string Picture { get; set; }

        public string Name { get; set; }

        public Boolean isOwner { get; set; }

        public string SenderId { get; set; }

        public string SenderEmail { get; set; }
    }
}
